gui/NumeReWindow.cpp

```diff
@@ -3526,6 +3531,11 @@ void NumeReWindow::EvaluateTab()
             return;
     }
 
+    if(m_book->getCurrentEditor()->HasBeenSaved() && m_book->getCurrentEditor()->getFileType() == FILE_NPRC || m_book->getCurrentEditor()->getFileType() == FILE_NLYT || m_book->getCurrentEditor()->getFileType() == FILE_NSCR)
+    {
+        m_terminal->UpdateLibrary();
+    }

```

Comment:
```suggestion

        if (m_book->getCurrentEditor()->getFileType() == FILE_NPRC 
            || m_book->getCurrentEditor()->getFileType() == FILE_NLYT 
            || m_book->getCurrentEditor()->getFileType() == FILE_NSCR)
        {
            m_terminal->UpdateLibrary();
        }
    }
```

Same thing as above.